Terminal.Gui/View/Layout/DimAuto.cs

```diff
@@ -15,50 +15,17 @@ namespace Terminal.Gui;
 ///         methods on the <see cref="Dim"/> class to create <see cref="Dim"/> objects instead.
 ///     </para>
 /// </remarks>
-public record DimAuto : Dim
+/// <param name="MaximumContentDim">The maximum dimension the View's ContentSize will be fit to. NOT CURRENTLY SUPPORTED.</param>
```

Comment:
The param tags are dual-purpose in a primary constructor for a record type. They show up both for the parameters of the constructor _as well as_ serve as the summary for the property they represent.